distribution/zip/jballerina-tools/build.gradle

```diff
@@ -114,6 +114,7 @@ dependencies {
     dist project(':ballerina-bindgen')
     dist project(':maven-resolver')
     dist project(':diagram-util')
+    dist project(':api-quoter')
```

Comment:
Here `api-quoter` is a generic term. Can we use `syntax-api-calls-gen`?

Reply:
+1 on the name. will change the module name and all the related classes to match this name